xilem_core/src/lib.rs

```diff
@@ -44,3 +44,7 @@ pub use any_view::AnyView;
 
 mod sequence;
 pub use sequence::{AppendVec, ElementSplice, ViewSequence};
+
+/// This alias is mostly syntax sugar to avoid the elaborate expansion of
+/// `<Self::Element as ViewElement>::Mut<'el>` in the View trait when implementing it (e.g. via rust-analyzer)
+pub type Mut<'el, E> = <E as ViewElement>::Mut<'el>;
```

Comment:
I'm surprised this works without a bound on `E: ViewElement`.

Reply:
Now that you're saying it, yeah I'm surprised as well ^^, should the bound be added, or should we leave it as is?

Reply:
I'd lean slightly towards adding it, although it doesn't really gain us too much either way

Reply:
Ok, when I add it, I'll get the `type_alias_bounds` warning, so, rust doesn't even want me to add a bound in the alias...